### ISSUE TITLE
macOS download info: File Quarantine

### ISSUE THREAD

**Issue by Liunkae** [brave/browser-laptop#13085]
The [File Quarantine](https://support.apple.com/en-us/HT201940) feature on macOS gives the user more information about the download (e.g., application that downloaded it, time of download, download webpage). Here's a screenshot:
![](https://user-images.githubusercontent.com/19424103/36043719-dd3d5580-0d95-11e8-9c44-6253c6f80388.png)

To implement this, the `LSFileQuarantineEnabled` needs to be added to the Brave app's Info.plist file. ([info](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/LaunchServicesKeys.html#//apple_ref/doc/uid/TP40009250-SW10))

Labels:  `suggestion`, `OS/macOS`, `security`, `sec-low`, `feature/download`

**Comment by Liunkae** [comment]
I tested this by manually adding the `LSFileQuarantineEnabled` key to my Brave app's Info.plist file, and it works as expected. Here's a screenshot after I tried to open a download from brave.com, downloaded by Brave:

![image](https://user-images.githubusercontent.com/19424103/36045699-33a3511c-0d9c-11e8-916b-8ffcf5a25972.png)

However, I'm not sure how to add this to the Brave app build process. Any help would be appreciated!
